src/mbgl/style/sources/geojson_source_impl.cpp

```diff
@@ -153,7 +153,7 @@ void GeoJSONSource::Impl::loadDescription(FileSource& fileSource) {
     });
 }
 
-Range<uint8_t> GeoJSONSource::Impl::getZoomRange() {
+Range<uint8_t> GeoJSONSource::Impl::getZoomRange() const {
     assert(loaded);
```

Comment:
As hinted at more generally in the second point of [this comment](https://github.com/mapbox/mapbox-gl-native/issues/6584#issuecomment-251744922), this assert is likely to get triggered if we start using `getZoomRange` more widely. It may need to return `optional<Range<uint8_t>>`, so that callers are aware of the possibility of absence.